### ISSUE TITLE
run_afl.sh file empty

### ISSUE THREAD

**Issue by elix0r** [julieeen/kleefl#2]
The run_afl.sh file is empty. Is this bash file no longer needed? 

`~/tests/libjpeg@master✗✗✗✗✗✗ #! cat /vagrant/tools/run_afl.sh`

`#!/bin/bash`

Thank you!

**Comment by julieeen** [comment]
Hi, the file is touched by _kleefl_init_ and will be setup by _kleefl_prepare_afl_ after KLEE is executed once. Finally the _run_afl.sh_ can be executed to start the fuzzers :)

**Comment by elix0r** [comment]
Makes sense. After running KLEE to generate the .bc/.ktext files I'm having issues getting the kleefl_prepare_afl to work (I moved the location of the .ktext files to be klee/klee-last). Please see output below:
```
~/test/project_1/tiff-3.8.2 #! ls fuzz             
in  run_afl.sh
~/test/project_1/tiff-3.8.2 #! python2 /vagrant/tools/kleefl_prepare_afl
Traceback (most recent call last):
  File "/vagrant/tools/kleefl_prepare_afl", line 472, in <module>
    main(sys.argv)
  File "/vagrant/tools/kleefl_prepare_afl", line 364, in main
    copy(afl_dir+"app", "app")
  File "/usr/lib/python2.7/shutil.py", line 119, in copy
    copyfile(src, dst)
  File "/usr/lib/python2.7/shutil.py", line 82, in copyfile
    with open(src, 'rb') as fsrc:
IOError: [Errno 2] No such file or directory: 'fuzz/app'
```

Is app supposed to be a subdirectory with the binary name? I have changed the script to create a subfolder under the fuzz directory which passes this issue, however the script fails later on. I'll see if I can get  the script to work as intended.

**Comment by julieeen** [comment]
Argh! I usually took the kleefl_pick utility to choose a binary for evaluation.
This will copy a target $app like /klee_build/$app.bc -> klee/app.bc and /afl_build/$app -> fuzz/app
This is missing in the README & has to be done before executing KLEE!
(What also explains why your klee-last folder doesn't have proper data & structure)

Thanks for sharing! 👍

**Comment by elix0r** [comment]
Awesome, I will add the documentation to my fork and send a merge request. Closing 👍 
Thanks again!!
